desktop/cmp/input/CodeInput.js

```diff
@@ -331,6 +332,13 @@ class Model extends HoistInputModel {
 
     toggleFullScreen() {
         this.modalSupportModel.toggleIsModal();
+
+        // 'Nudge' the mouse wheel to trigger CodeMirror to update scrollbar state
+        const scrollEvent = d => new window.WheelEvent('mousewheel', {deltaX: d, deltaY: d});
+        wait().then(() => {
+            this.editor.getScrollerElement().dispatchEvent(scrollEvent(2));
+            this.editor.getScrollerElement().dispatchEvent(scrollEvent(-2));
+        });
```

Comment:
This is an interesting workaround! Did you try `this.editor.refresh()`? It seems to me from the docs that that might do the trick in a simpler way.

Reply:
`this.editor.refresh()` doesn't seem to refresh the scrollbars unfortunately.